data/ListingsData.json

```diff
@@ -138,6 +152,6 @@
     "RequestedUserIDs": [
       "448"
     ],
-    "Id": "7646"
+    "Id": null
```

Comment:
Id shouldn't be null

Reply:
This shouldn't be null, didn't change it manually, may be a bug.